javascript/core/intro-js/javascript-basics-part-2.md

```diff
@@ -26,68 +15,79 @@ aspects:
 ---
 ## Content
 
-Like most programming languages, JavaScript has a number of syntax rules that must be adhered to.
+Like most programming languages, JavaScript has a number of syntax rules that it must adhere to.
 
 ### Semicolon
 
-In ECMAScript it is best practice to add a semicolon at the end of a statement:
+Statements in JavaScript end in a `;`. Based on the ECMAScript[1] rules however, adding a `;` isn't actually required (although it is common practice):
```

Comment:
Just a suggestion.
`Isn't actually required` can be shortened to just `isn't required`.